com.github.Resticity.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.Resticity</id>
```

Comment:
this app-id doesn't comply with the requirement. As it is github based it should be `io.github` and have four components with the third one being the user/org id from github.

Reply:
Hi, thx for your reply... I remember I had some issue elsewhere, because my username contains dashes. So I thought I need to simplify it down without the username.

Reply:
the documentation say what to do in this case.